### ISSUE TITLE
Scene stacking with camera fade throws WebGL buffer overflow

### ISSUE THREAD

**Issue by vinerz** [photonstorm/phaser#4852]
## Version
* Phaser Version: 3.20.1
* Operating system: macOS Catalina 10.15.0
* Browser: Chrome 78

## Description

Stacking a `Scene` above another `Scene` which uses a `BitmapMask` and then fading the top `Scene` camera generates **WebGL buffer overflow** indefinitely.

## Example Test Code

```js
class Scene1 extends Phaser.Scene {
    constructor() {
        super({ key: 'Scene1', active: true });
    }

    preload() {
        this.load.image('bunny', 'assets/sprites/bunny.png');
        this.load.image('phaser2', 'assets/sprites/phaser2.png');
        this.load.image('checker', 'assets/pics/checker.png');
    }

    create() {
        var bunny = this.make.tileSprite({
            x: game.config.width / 2, 
            y: game.config.height / 2,
            width: 400,
            height: 500,
            key: 'bunny',
            add: true
        });

        bunny.blendMode = Phaser.BlendModes.ADD;

        var phaser2 = this.make.image({
            x: game.config.width / 2,
            y: game.config.height / 2,
            key: 'phaser2',
            add: false
        });

        bunny.mask = new Phaser.Display.Masks.BitmapMask(this, phaser2);

        this.input.on('pointermove', function (pointer) {
            phaser2.x = pointer.x;
            phaser2.y = pointer.y;
        });
    }
}

class Fader extends Phaser.Scene {
  constructor() {
    super({ key: 'Fader', active: true });
  }

  create() {
    this.cameras.main.transparent = true;
    this.cameras.main.fadeOut(0);
  }
}

var config = {
    type: Phaser.WEBGL,
    parent: 'phaser-example',
    width: 640,
    height: 480,
    scene: [
        Scene1,
        Fader,
    ]
};

var game = new Phaser.Game(config);
```

## Additional Information

- If you try to flash the camera, the same error is thrown and the flash takes no effect.
- If you don't fade the camera, the error doesn't happen.
- If the fading camera `Scene` is below the Scene with `BitmapMask`, the error also doesn't happen.

**Comment by vinerz** [comment]
**Interesting and oddly specific discover:** Drawing a transparent single pixel at a `Graphics GameObject`, which has the highest `depth` at the `Scene`, at the last instruction inside the `update` method, seems to resolve this issue.

**Comment by photonstorm** [comment]
Just to say this was fixed in the 3.23 release, sorry for not updating this issue.
